daq/docker_test.py

```diff
@@ -18,13 +20,15 @@ class DockerTest:
     TAGGED_IMAGE_FORMAT = IMAGE_NAME_FORMAT + ':latest'
     CONTAINER_PREFIX = 'daq'
 
-    # pylint: disable=too-many-arguments
-    def __init__(self, host, target_port, tmpdir, test_name, module_config):
-        self.target_port = target_port
+    def __init__(self, host, tmpdir, test_name, module_config):
+        self.device = host.device
         self.tmpdir = tmpdir
         self.test_name = test_name
         self.runner = host.runner
-        self.host_name = '%s%02d' % (test_name, self.target_port)
+        # Host name can't be more than 15 characters.
+        # The name is also used to create interface in mininet
```

Comment:
Are these two statements logically connected?  If so, add a "because" in there to link them, otherwise the second commend doesn't have much context.